### DISCUSSION TITLE
Friends list

### DISCUSSION THREAD

**Discussion by iloveorcas27** [monkeytypegame/monkeytype#939] [Ideas] [answered]
It would be nice if there was a sort of friends list, where you could "friend" other people on Monkeytype and see each other's stats.

**Comment by Miodec** [comment, marked as answer]
Its not in the plans currently, maybe in the future.

**Reply by ObiKeahloa** [reply]
Would be nice if it is also possible that anyone can see your stats (As in without being friended ) or make your stats private if it where to be added.
